test/filters/test_sepia.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils
+from kornia.testing import assert_close
+
+
+@pytest.mark.parametrize(
+    "batch_shape,expected",
+    [
+        ((2, 3, 1, 1), [[[[1.0]], [[0.8905]], [[0.6936]]], [[[1.0]], [[0.8905]], [[0.6936]]]]),
+        ((3, 1, 1), [[[1.0]], [[0.8905]], [[0.6936]]]),
+    ],
+)
+def test_sepia_calc(batch_shape, expected, device, dtype):
+    input_tensor = torch.ones(batch_shape, device=device, dtype=dtype)
+    expected_tensor = torch.tensor(expected, device=device, dtype=dtype)
+
+    actual = kornia.filters.sepia(input_tensor, rescale=True)
+    assert_close(actual, expected_tensor, rtol=1e-4, atol=1e-4)
+
+
+def test_sepia_calc_without_rescale(device, dtype):
+    input_tensor = torch.tensor([[[0.1, 0, 1]], [[0.1, 0, 1]], [[0.1, 0, 1]]], device=device, dtype=dtype)
+
+    expected = torch.tensor(
+        [[[0.1351, 0.0, 1.3510]], [[0.1203, 0.0, 1.2030]], [[0.0937, 0.0, 0.9370]]], device=device, dtype=dtype
+    )
+
+    actual = kornia.filters.sepia(input_tensor, rescale=False)
+    assert_close(actual, expected)
+
+
+def test_sepia_uint():
+    input_tensor = torch.tensor([[[10, 0, 255]], [[10, 0, 255]], [[10, 0, 255]]], dtype=torch.uint8)
+
+    expected = torch.tensor([[[112, 0, 168]], [[224, 0, 208]], [[76, 0, 18]]], dtype=torch.uint8)
+
+    actual = kornia.filters.sepia(input_tensor, rescale=False)
+    assert_close(actual, expected)
+
+
+class TestSepia:
```

Comment:
```suggestion
class TestSepia(BaseTester):
```
worth to start using https://github.com/kornia/kornia/blob/master/kornia/testing/__init__.py#L85

/cc @MrShevan

Reply:
Done at https://github.com/kornia/kornia/pull/1947/commits/88dc8d1f280f49b8d87d8e96acb8581383aae7d4